### PR TITLE
Fixed the line endings to accommodate different git configurations independent of the platform.

### PR DESCRIPTION
**Pull request by enternoescape**

### CODE REVIEW

File: test/java/sage/IOUtilsTest.java [hunk 1]
```diff
@@ -14,14 +14,26 @@
   public void testGetUrlAsString() throws Exception
   {
     String md5 = IOUtils.getUrlAsString(file2url(getTestResource("test-jar-1.0.zip.md5")));
-    assertEquals(md5, "beec1608cf997d3acb42f0ab772b143f" + System.lineSeparator());
+    assertNotNull(md5);
```
Comment (Narflex) [inline test/java/sage/IOUtilsTest.java]: Why not just use String.trim() on the MD5 values and then compare them after doing that instead?

Comment (enternoescape) [inline test/java/sage/IOUtilsTest.java]: I guess that would have the same effect. I was thinking what if more than what we were expecting was read for some reason. Trim might remove a real problem.

**Comment by Narflex** [comment]
Oh sorry...my bad. I thought it was doing an MD5 comparison because it was referencing MD5 values...I didn't realize this was testing getFileAsString and getURLAsString...in that case, then I'd think there should be no difference at all per-platform relating to line endings. I'll take a look at this and likely submit another change. :)

**Comment by Narflex** [comment]
OK, the problem is that the GIT client may end up modifying the line endings on the test data file when it is downloaded...that's why it can have \r\n or \n as the line ending.  So now I actually prefer your original fix then. :)  I'll revert it back to what you originally submitted...my bad.

**Comment by Narflex** [comment]
Reversion back to your original fix submitted.

**Comment by enternoescape** [comment]
Now worries. Thanks for the laugh. :)

**Pull request merged**